ui/page/staking/overview.go

```diff
@@ -36,17 +39,17 @@ type Page struct {
 	toTickets           decredmaterial.TextAndIconButton
 
 	stakingOverview *dcrlibwallet.StakingOverview
-	liveTickets     []*transactionItem
+	liveStakes      []*transactionItem
```

Comment:
```suggestion
	liveTickets      []*transactionItem
```
I believe live tickets is still very appropriate here, the only reason the button title was changed to stake was because you don't really purchase tickets if the money is returned back to you